rqt_bag/src/rqt_bag/bag_timeline.py

```diff
@@ -729,7 +752,8 @@ def _message_recorded(self, topic, msg, t):
                 try:
                     listener.timeline_changed()
                 except Exception as ex:
-                    qWarning('Error calling timeline_changed on %s: %s' % (type(listener), str(ex)))
+                    listen_type = type(listener)
+                    qWarning('Error calling timeline_changed on %s: %s' % (listen_type, str(ex)))
```

Comment:
You're saving a single character with the assignment. Might as well use an f-string and keep it on one line.
```suggestion
                    qWarning(f'Error calling timeline_changed on {type(listener)}: {str(ex)}')
```